dotnet/dotnet-cloud-run-hello-world/.vscode/tasks.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Is this file meant to be generated? It's unclear to me what we want to include here.

Reply:
Yes, it should but the content depends on the language.
For .net the goal was to have tasks for building and packaging of the code locally. It can contain other tasks that are common, such as running unit tests, if it's applicable.